src/components/body/main/Main.tsx

```diff
@@ -0,0 +1,37 @@
+import { Route, Routes } from "react-router-dom"
+import Lessoner from "./lessoner/Lessoner"
+import Categories from "./categories/Categories"
+import Lessons from "./lessons/Lessons"
+import About from "./about/About"
+import Sign_in from "./Sign_in/Sign_in"
+import Sign_up from "./lessoner/Sign_up/Sign_up"
+import "./Main.css"
+import Link1 from "../../Link1/Link1"
+import Link2 from "../../Link2/Link2"
+import Link3 from "../../Link3/Link3"
+
+
+
+
+const Main: any = () => {
+
+
+
+    return (
+        <div className="main">
+            <Routes>
+                <Route path="/link1" element={<Link1/>}/>
+                <Route path="/link2" element={<Link2/>}/>
+                <Route path="/link3" element={<Link3/>}/>
+                <Route path="/" element={<Lessoner  />} />
+                <Route path="/categories" element={<Categories />} />
+                <Route path="/lessons" element={<Lessons />} />
+                <Route path="/about" element={<About />} />
+                <Route path="/users/sign_in" element={<Sign_in/>}/>
+                <Route path="/users/sign_up" element={<Sign_up/>} />
```

Comment:
remove unnecessary routes, leave only link to the ExamplePage 

upd: or, with according to the ticket - to the: main page (root page), categories, lessons, portfolio